pucauto.py

```diff
@@ -46,6 +46,7 @@ def confirm_trade():
 
 def main():
     login()
+    assert 'pucatrade.com/dashboard' in browser.title
```

Comment:
There is no `browser.title`. You'd want `driver.title` instead. However, it looks like you're asserting on the url. The title will be "Dashboard |  PucaTrade - Trade Magic: The Gathering Cards Online".

So you can use `driver.current_url` for this assertion.